runtime/lua/vim/treesitter/dev.lua

```diff
@@ -436,11 +457,153 @@ function M.inspect_tree(opts)
     buffer = buf,
     once = true,
     callback = function()
-      if api.nvim_win_is_valid(w) then
-        api.nvim_win_close(w, true)
+      close_win(w)
+    end,
+  })
+end
+
+-- between the query editor and the tree inspector.
```

Comment:
What happened to this sentence?

Reply:
Oops this was part of a (now gone) `TODO`: 1e6da4ad1e6112cb964267e69132fecce098ae35